### PR TITLE
[IMP] l10n_cg,l10n_syscohada: activate taxes and fix some accounts

### PR DESCRIPTION
**Pull request by MuhamedAbdallaMuhamed**
For `l10n_cg`, some taxes are unarchived.

For `l10n_syscohada`, the type of some accounts is changed from Equity
to Current liabilities or Current Asset.
A missing account is also added.

task-3848455

### CODE REVIEW

**Review by aboo-odoo — COMMENTED** [review]
The community part seems good, well done :smile: 

But the same comment about commit message and PR description applies :+1:

**Comment by clbr-odoo** [comment]
I suppose that's for LABE/OCO's scope now :)

**Review by oco-odoo — CHANGES_REQUESTED** [review]
We should split this PR into two commits IMO

File: addons/l10n_cg/data/template/account.tax-cg.csv [hunk 1]
```diff
@@ -17,19 +17,19 @@
...
 "","","","","","","","","","","base","refund","-purc_exempt","","","",""
 "","","","","","","","","","","tax","refund","","","","",""
-"tva_sale_18","18%","","False","","18.0","percent","sale","tax_group_18","","base","invoice","+base_18","","","",""
-"","","","","","","","","","","tax","invoice","+tax_18","pcg_4431","","",""
+"tva_sale_18","18%","","True","","18.0","percent","sale","tax_group_18","","base","invoice","+base_18","","","",""
+"","","","","","","","","","","tax","invoice","+tax_18","","","",""
 "","","","","","","","","","","base","refund","-base_18","","","",""
-"","","","","","","","","","","tax","refund","-tax_18","pcg_4431","","",""
-"tva_purchase_18","18%","","False","","18.0","percent","purchase","tax_group_18","","base","invoice","+purc_base","","","",""
-"","","","","","","","","","","tax","invoice","+purc_tax","pcg_4452","","",""
```
[the remark this answers is not in the document]
Comment (Arnaud-Sibille) [reply addons/l10n_cg/data/template/account.tax-cg.csv:25]: To be clear, before your comment I only wrote the commit msg, so it's me who just forgot to mention the accounts changes ;)

But anyway, I've seperated into 2 commits and removed the accounts changes, as discussed

**Comment by oco-odoo** [comment]
@robodoo rebase-ff

**Review by oco-odoo — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**